data/meta/oss_fuzz_python.jsonl

```diff
@@ -1,264 +1,264 @@
-{"homepage": "https://github.com/hynek/pem", "language": "python", "main_repo": "https://github.com/hynek/pem", "fuzzing_engines": ["libfuzzer"], "sanitizers": ["address"], "vendor_ccs": ["david@adalogics.com", "adam@adalogics.com", "arthur.chan@adalogics.com"], "#fuzz_target": 1, "repo_id": "hynek/pem"}
+{"fuzzing_engines": ["libfuzzer"], "homepage": "https://github.com/abseil/abseil-py", "language": "python", "main_repo": "https://github.com/abseil/abseil-py", "sanitizers": ["address", "undefined"], "vendor_ccs": ["david@adalogics.com"], "#fuzz_target": 1, "repo_id": "abseil/abseil-py"}
```

Comment:
please don't modify this file, suggest do `git restore data/meta/oss_fuzz_python.jsonl`